ui/src/app/pages/login/account-creation/account-creation.tsx

```diff
@@ -409,14 +409,26 @@ export class AccountCreation extends React.Component<AccountCreationProps, Accou
         };
       }
     }
-    return validate(validationData, validationCheck);
+
+    const urlError = validationData.professionalUrl
+      ? validate({website: validationData.professionalUrl}, {
+        website: { url: { message: '^Professional URL %{value} is not a valid URL' } }
+      })
+      : undefined;
+
+    const errors = {
+      ...validate(validationData, validationCheck),
+      ...urlError
+    };
+
+    return fp.isEmpty(errors) ? undefined : errors;
   }
 
   render() {
     const {
       profile: {
         givenName, familyName,
-        contactEmail, username, areaOfResearch, professionalUrl,
+        contactEmail, username, areaOfResearch, professionalUrl = '',
```

Comment:
Defaulting `professionalUrl` prevents a react error from showing up in the console from having an input field initialized as undefined